tests/dragonfly/__init__.py

```diff
@@ -162,7 +162,8 @@ def __str__(self):
 
     @property
     def dfly_path(self):
-        return str(os.path.dirname(self.params.path))
+         path = os.path.dirname(self.params.path)
+         return str(os.path.join(path, ''))
```

Comment:
On the previous comment I meant that you should use the os.path.join when you already have the filename